### PR TITLE
[ICP-11752]fix remain battery percentage incorrect issue

### PR DESCRIPTION
**Pull request by FenMei007S**
@tpmanley

### CODE REVIEW

**Comment by FenMei007S** [comment]
@tpmanley , plz  review it

**Comment by FenMei007S** [comment]
okay,Happy Thanksgiving Day^^

File: devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy [hunk 1]
```diff
@@ -84,7 +84,11 @@ def parse(String description) {
 	if (!map) {
 		Map descMap = zigbee.parseDescriptionAsMap(description)
 		if (descMap.clusterInt == 0x0001 && descMap.commandInt != 0x07 && descMap?.value) {
-			map = getBatteryResult(Integer.parseInt(descMap.value, 16))
+			if (descMap.attrInt == 0x0021){
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy]: Can you add a space before the `{` and fix the indentation below?

**Comment by tpmanley** [comment]
@FenMei007S this does look safe but we'll need to hold it back from the release today because it doesn't fit our criteria for a fast track release. We'll merge this tomorrow after you make the minor edits requested.

**Comment by FenMei007S** [comment]
Update

**Review by tpmanley — APPROVED** [review]

**Pull request merged**